src/riak_kv_continuation.erl

```diff
@@ -0,0 +1,118 @@
+%% -------------------------------------------------------------------
+%%
+%% riak_kv_continuation: creating and decoding continuation tokens
+%%
+%% Copyright (c) 2017 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+%% @doc Central module for creating and decoding continuation tokens.
+
+-module(riak_kv_continuation).
+
+-export([make_token/1,
+         make_token/2,
+         make_token/3,
+         decode_token/1]).
+-export_type([token/0]).
+
+-type token() :: binary() | undefined.
+
+%% @doc Make a continuation token for the given `Term'.
+-spec make_token(term()) -> token().
+make_token(Term) ->
```

Comment:
I wonder if it might be worth adding an extra byte to the beginning of the token to denote format/version, in case we decide to change to something else later. If nothing else, it'd be nice for efficiency if we could base64-encode raw binaries directly instead of having to wrap them in the extra layer of `term_to_binary`.

On the flip side, this might be overkill though, since I'm not sure if tokens are something we'll be using heavily enough to warrant this level of optimization. I guess use your own judgement :)